po/es.po

```diff
@@ -1210,12 +1210,12 @@ msgstr "¡No tiene ninguna cuenta de provisión con balances!"
 #: gnucash/gnome/dialog-invoice.c:1589
 #: gnucash/gnome-utils/gnc-general-select.c:220
 msgid "Select..."
-msgstr "Seleccionar…"
+msgstr "Seleccionar..."
```

Comment:
@cyphra Wrong direction! We want to use `…` in the future, but older compilers did not allow it before 4.0